# HG changeset patch
# User Jacob Dawid <jacob.dawid@gmail.com>
# Date 1342791464 14400
#      Fri Jul 20 09:37:44 2012 -0400
# Node ID 2318531b7880ec1a5824a51157fc28434617f42b
# Parent  76452c97add8ccdb0794dd3c0ca82dfdd429b33d
Changed cursor to underline cursor.

diff --git a/libqterminal/unix/QUnixTerminalImpl.cpp b/libqterminal/unix/QUnixTerminalImpl.cpp
--- a/libqterminal/unix/QUnixTerminalImpl.cpp
+++ b/libqterminal/unix/QUnixTerminalImpl.cpp
@@ -28,16 +28,17 @@ QUnixTerminalImpl::QUnixTerminalImpl(QWi
     : QTerminalInterface(parent) {
     setMinimumSize(600, 400);
     initialize();
 }
 
 void QUnixTerminalImpl::initialize()
 {
     m_terminalView = new TerminalView(this);
+    m_terminalView->setKeyboardCursorShape(TerminalView::UnderlineCursor);
     m_terminalView->setBellMode(TerminalView::NotifyBell);
     m_terminalView->setTerminalSizeHint(true);
     m_terminalView->setContextMenuPolicy(Qt::CustomContextMenu);
     m_terminalView->setTripleClickMode(TerminalView::SelectWholeLine);
     m_terminalView->setTerminalSizeStartup(true);
     m_terminalView->setSize(80, 40);
     m_terminalView->setScrollBarPosition(TerminalView::ScrollBarRight);
 
